haproxy-ingress/templates/_podtemplate.yaml

```diff
@@ -65,8 +65,18 @@ spec:
         - --{{ $key }}
   {{- end }}
 {{- end }}
+{{- if or (not .Values.controller.haproxy.enabled) .Values.controller.metrics.enabled }}
+      ports:
+  {{- if not .Values.controller.haproxy.enabled }}
+        {{- include "haproxy-ingress.controller.ports" . | nindent 8 }}
```

Comment:
This include need to be unindented in two spaces, it's breaking the controller-metrics below if declared. Or the ports template could be unindented instead.

Reply:
I decided to unindent the ports template, it's better that no indentation is included and instead set it where the template is used.